ruby26/plan.sh

```diff
@@ -39,6 +39,6 @@ do_check() {
 
 do_install() {
   do_default_install
-  gem update --system --no-document
+  gem update --system 2.7.9 --no-document
```

Comment:
@jonsmorrow It's been a while since I've actively use Ruby, but my understanding was that we want Gem >=3.0 for Ruby 2.6.    We're currently shipping 3.0.3 in the ruby26 package.  Is this correct that we want 2.7.9 for Ruby 2.6?  What are the ramifications of a downgrade?

Reply:
That Idk I assumed incorrectly that you meant 2.6 when you said 2.4 above. 2.4 pulls it's build steps from the main plan so when I saw this unpinned here I thought you mistyped. I'll put this back.

Reply:
Sorry I wasn't clear.  I _think_ we want  `ruby`, `ruby24` and `ruby25` to all be on `gem 2.7.X`, and `ruby26` to be on `gem 3.X.X`.   

That's my understanding of the Ruby/Gem ecosystem and versioning,  but please feel free to correct me if I'm wrong.